package.json

```diff
@@ -25,8 +25,9 @@
     "publish-release": "branch=$(git rev-parse --abbrev-ref HEAD) && [ $branch = main ] && yarn lerna publish --conventional-graduate --exact || echo 'Error: New release can only be published on main branch' >&2",
     "publish-prerelease": "branch=$(git rev-parse --abbrev-ref HEAD) && [ $branch = main ] && yarn lerna publish prerelease --preid beta --dist-tag next --exact || echo 'Error: New release can only be published on main branch' >&2",
     "test": "jest",
-    "test-cli": "lerna run test-headless --stream",
-    "jest": "jest --watch",
+    "test-cli": "cd acceptance-tests && yarn test-headless --stream",
+    "test-docker": "yarn build-docker && docker container run -it --rm --name=hs-cli-container hs-cli-image yarn test",
```

Comment:
This command will start up the container based off of the created image. It will then run the tests inside of it, and then remove itself after completion.